docs/redux.md

```diff
@@ -0,0 +1,118 @@
+# Redux configuration
+
+## What is Redux
+
+Flex Template for Web (FTW) is JavaScript single-page application (SPA). This means that the app
+needs to be able to render several different layouts (pages) depending on user interaction. State
+management is essential for this process. FTW needs to know if a user has been authenticated, if it
+has received relevant data for the current page, and so on.
+
+We use [Redux](https://redux.js.org/introduction) for state management on the application level. You
+should read more about Redux before you start modifying queries to Flex API or creating new Page
+level elements (unless you are modifying [a static page](static-pages.md)).
+
+In the following subtopics, we assume that you know the
+[basics of Redux](https://redux.js.org/basics) already.
+
+* [Containers: Pages + TopbarContainer](#containers-pages--topbarcontainer)
+* [Duck files](#duck-files)
+* [Setting up Redux](#setting-up-redux)
+* [Advanced Redux concepts: thunks](#advanced-redux-concepts-thunks)
+
+## Containers: Pages + TopbarContainer
+
+We have set up FTW so that pages are aware of Redux, but other components and forms are purely
+presentational components. This makes it easier to customize UI components - you don't need to be
+aware of the complexity related to Redux setup when you just want to touch the behavior of some
+visual component. In those cases, you can just head to `scr/components/` or `src/forms/` folder and
+you can see from there what props are available for each component when they are rendered.
+
+Naturally, there is a need for higher level components which fetch new data and define what props
+are passed down to presentational components. In Redux terminology, those components are called
+_Containers_. FTW has defined all the containers inside a folder called `src/containers/`. It
```

Comment:
Ok, here is part of the explanation for "containers". Maybe add something earlier for both "presentational" and "containers"?

Reply:
I added a link to the first occurrence of "presentational component" and another "read more" pointer to the end of this paragraph.